src/main/java/com/fasterxml/jackson/databind/DeserializationContext.java

```diff
@@ -161,21 +161,11 @@ public abstract class DeserializationContext
     /**********************************************************
      */
 
-    protected DeserializationContext(DeserializerFactory df) {
```

Comment:
I decided to remove this constructor to ensure non-null `DeserializerCache` is passed from sub-classes.
Should not be problematic as extension point is `DefaultDeserializationContext`.